frontend/lib/src/components/shared/AlertContainer/AlertContainer.tsx

```diff
@@ -98,7 +99,9 @@ export default function AlertContainer({
         },
       }}
     >
-      <StyledAlertContent>{children}</StyledAlertContent>
+      <StyledAlertContent data-testid={`stNotificationAlertContent${testid}`}>
```

Comment:
Prefer the idea of `stNotification${testid}Content` for simplicity - so it would be "stNotificationInfoContent" or "stNotificationSuccessContent". WDYT?

Reply:
Hmm what about `stNotificationContent${testid}` ? I like reading left to right and then seeing the components. The ending is kind of just extra information.

Reply:
Makes sense - good with either `stNotificationContent${testid}` or `stNotificationContent-${testid}`? since we've used the dash to separate the component from the extra info before

Reply:
going to go with `stNotificationContent${testid}`. I was thinking we should use the `-` for duplicate words. Might need to think about that more.